lib/v2/sspai/radar.js

```diff
@@ -42,7 +42,7 @@ module.exports = {
                 title: '作者已发布文章',
                 docs: 'https://docs.rsshub.app/routes/new-media#shao-shu-pai-sspai',
                 source: '/u/:id/posts',
-                target: '/sspai/author/:id',
+                target: '/sspai/author/:id/:keyword?',
```

Comment:
`keyword` is always empty here since it wasn't matched in `source`

Reply:
i have revert this to previous version